libqtile/widget/statusnotifier.py

```diff
@@ -484,17 +484,35 @@ def __init__(self):
         self.items: List[StatusNotifierItem] = []
         self.name = "qtile"
         self.icon_theme: str = None
+        self.started = False
+        self.on_item_added: List[Callable] = []
+        self.on_item_removed: List[Callable] = []
+        self.on_icon_changed: List[Callable] = []
 
```

Comment:
If these are private attributes (not accessed by your code in qtile-extras) could we prefix with underscores so that's clear in the future?